designer_v2/lib/features/design/study_form_controller.dart

```diff
@@ -151,7 +151,7 @@ class StudyFormViewModel extends FormViewModel<Study> implements IFormViewModelD
   Future _applyAndSaveSubform(IStudyFormData subformData) {
     studyDirtyCopy ??= formData!.exactDuplicate();
     subformData.apply(studyDirtyCopy!);
-    // Flush the on-write study copy to the repository & clear it
+    // Flush the on-write study copy to the repository and clear it
```

Comment:
There are quite a few other places with `&` in comments; if this is relevant we should consider changing all of them